src/wx/cmdevents.cpp

```diff
@@ -869,7 +878,7 @@ EVT_HANDLER_MASK(SaveDotCodeFile, "Save e-Reader Dot Code...", CMDEN_GBA)
         return;
 
     savedotcodefile_path = dlg.GetPath();
-    SetSaveDotCodeFile(savedotcodefile_path.mb_str(wxConvUTF8));
+    SetSaveDotCodeFile(savedotcodefile_path.mb_str());
```

Comment:
@s520 Just for clarification, were the issues regarding the path being garbled on Windows?

Reply:
Yes. I confirmed it in Japanese environment of Windows 10 64bit.
I don't know wxWidgets in detail, but if you specify wxConvUTF8 as an argument, this part will be garbled if the path contains Japanese (multibyte characters?).

Reply:
Thank you! We have some bugs about converting between UTF16 and UTF8 for windows, so I just wanted to confirm this was part of it. Nevertheless, this should be enough for now.